src/Core/Telemetry/ITelemetryManager.cs

```diff
@@ -28,5 +28,7 @@ public interface ITelemetryManager
         void Update();
         void LanguageAnalyzed(string languageKey);
         void ShowHotspotRequested();
+        void TaintIssueOpened();
```

Comment:
As we discussed online, it might be time to switch to a generic method `void IncrementDailyCounter(string counterName)`, which would reduce a lot of the copy-paste. We can do it next time we need to add telemetry.